packages/beacon-node/src/chain/bls/multithread/jobItem.ts

```diff
@@ -70,7 +70,7 @@ export function jobItemWorkReq(job: JobQueueItem, metrics: Metrics | null): BlsW
       // and not a problem in the near future
       // this is monitored on v1.21.0 https://github.com/ChainSafe/lodestar/pull/6894/files#r1687359225
       const timer = metrics?.blsThreadPool.aggregateWithRandomnessMainThreadDuration.startTimer();
-      const {pk, sig} = aggregateWithRandomness(job.sets.map((set) => ({pk: set.publicKey, sig: set.signature})));
+      const {pk, sig} = await asyncAggregateWithRandomness(job.sets.map((set) => ({pk: set.publicKey, sig: set.signature})));
       timer?.();
```

Comment:
I'm not sure how to make this metric more useful, if not just remove it

Reply:
I updated the metric to reflect libuv time and added one for number of sets total.  Wondering if maybe we should make that one a histogram to see number of sets per job though.  What do you think @twoeths?

Reply:
I don't see the benefit due to `async/await` nature. If there are multiple async calls right after they are still counted in this metric
maybe we should record length of `job.sets` instead? and the count of `asyncAggregateWithRandomness()` calls would be helpful

Reply:
We already collect that.  I removed them from around the asyncAggregateWithRandomness.

https://github.com/ChainSafe/lodestar/blob/mkeil/async-aggregate-with-randomness/packages/beacon-node/src/chain/bls/multithread/index.ts#L429-L432